plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenCoordinatesUi.java

```diff
@@ -99,10 +104,13 @@ public String getVersion() {
     return versionField.getText().trim();
   }
 
-  public void addModifyListener(ModifyListener listener) {
-    groupIdField.addModifyListener(listener);
-    artifactIdField.addModifyListener(listener);
-    versionField.addModifyListener(listener);
+  public void addChangeListener(Listener listener) {
```

Comment:
I also need to add `PageValidator` (currently implementing `ModifierListener`) to the master control button for triggering validation when the value changes, but the button does not accept `ModifierListener`. Instead it accepts `SelectionListener`. So I decided to accept the generalized `Listener` version.